backend/packages/Upgrade/src/api/controllers/ExperimentUserController.ts

```diff
@@ -71,15 +78,19 @@ export class UserController {
    *       responses:
    *          '200':
    *            description: Get user By Id
+   *          '400':
+   *            description: BadRequestError - InvalidParameterValue
+   *          '401':
+   *            description: AuthorizationRequiredError
    *          '404':
-   *            description: user not found
+   *            description: Experiment User not defined
    */
   @Get('/:id')
-  @OnUndefined(UserNotFoundError)
-  public one(@Param('id') id: string, @Req() request: AppRequest): Promise<ExperimentUser> {
-    if (!isUUID(id)) {
-      return Promise.reject(new Error(SERVER_ERROR.INCORRECT_PARAM_FORMAT + ' : id should be of type UUID.'));
-    }
+  public async one(
+    @Params({ validate: true }) { id }: IdValidator,
+    @Req() request: AppRequest
+  ): Promise<ExperimentUser> {
+    await this.experimentClientController.checkIfUserExist(id, request.logger);
```

Comment:
Okay, so for find queries we should show 200 success with a message that user not found and not throw a 404 error?

Reply:
These `/experimentusers` endpoints are not included in the new automated testing suite, so there's no behavior defined there. That said, I think it would be consistent to return a 404 if no user is found. What are these endpoints used for?

Reply:
Okay, I see that these are not getting used in the frontend side. Probably they were used for development testing maybe. @VivekFitkariwala Do you remember the use for these api's?